### ISSUE TITLE
Aligning Videos in Space and Time

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#70]
# TL;DR
A study of unsupervised detection of behavioral correspondences in video. Learning patch-level correspondence by measuring the distance between patches detected in unsupervised training trackers. The detection accuracy is better than that of existing feature extractors.
![sCo (V4), fo (W,))](https://user-images.githubusercontent.com/24524018/92557350-fe1c9380-f2a6-11ea-83fc-78a6f882a658.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2007.04515

# Submission Dates(yyyy/mm/dd)
2020/07/09

# Authors and institutions
Senthil Purushwalkam, Tian Ye, Saurabh Gupta, Abhinav Gupta
- Carnegie Mellon University
- University of Illinois at Urbana-Champaign
- Facebook AI Research

# Methods

# Results

![Query Video](https://user-images.githubusercontent.com/24524018/92557359-0379de00-f2a7-11ea-9c1b-ce3f2c233e90.png)

# Comments
ECCV2020
